README.md

```diff
@@ -1,148 +1,43 @@
-
-``` purescript
-type URIRef userInfo hosts path hierPath relPath query fragment = ...
-```
-
-This allows us to provide hooks into the parsing and printing processes for a URI, so that types better suited to the intended use case can be used.
-
-Taking `userInfo` as an example, according to the spec, the `user-info` part of an authority is just an arbitrary string of characters terminated by an `@` before a hostname. An extremely common usage for this is the `user:password` scheme, so by leaving the choice of representation as a type variable we can switch it out for a type specifically designed to handle that (this library includes one actually, under [`URI.Extra.UserPassInfo`][UserPassInfo]).
-
-### App-specific URI type definitions
-
-When using this library, you'll probably want to define type synonyms for the URIs that make sense for your use case. A URI type that uses the simple representations for each component will look something like this:
-
-``` purescript
-type MyURI = URIRef UserInfo (HostPortPair Host Port) Path HierPath RelPath Query Fragment
-```
-
-Along with these types, you'll want to define an options record that specifies how to parse and print URIs that look like this:
-
-``` purescript
-options ∷ Record (URIRefOptions UserInfo (HostPortPair Host Port) Path HierPath RelPath Query Fragment)
-options =
-  { parseUserInfo: pure
-  , printUserInfo: identity
-  , parseHosts: HostPortPair.parser pure pure
-  , printHosts: HostPortPair.print identity identity
-  , parsePath: pure
-  , printPath: identity
-  , parseHierPath: pure
-  , printHierPath: identity
-  , parseRelPath: pure
-  , printRelPath: identity
-  , parseQuery: pure
-  , printQuery: identity
-  , parseFragment: pure
-  , printFragment: identity
-  }
-```
-
-As you can see by all the `pure` and `identity`, we're not doing a whole lot here. `parseHosts` is a bit of an exception, but that's just due to the way that case is handled (see [later in this README](#host-parsing) for more details about that).
-
-These types ([`UserInfo`][UserInfo], [`HostPortPair`][HostPortPair], [`Host`][Host], etc.) are all provided by the library, and where necessary can only be constructed via smart constructor. This ensures that percent-encoding is applied to characters where necessary to ensure the constructed values will print as valid URIs, and so on.
-
-If we decided that we wanted to support `user:password` style user-info, we'd modify this by changing our type to use [`UserPassInfo`][UserPassInfo]:
-
-``` purescript
-type MyURI = URIRef UserPassInfo (HostPortPair Host Port) Path HierPath RelPath Query Fragment
-```
-
-And update our options to use the appropriate parse/print functions accordingly:
+Install `purescript-uri` with [Spago](https://github.com/purescript/spago):
```

Comment:
The package name should be `uri`, in keeping with the package name as recorded in the registry and what's kept in the `spago.dhall` file. When using the contrib-updater tool you can just leave off the `--display-name` flag to have it use the package name as the display name.